packages/utils/scaffold.js

```diff
@@ -38,7 +38,7 @@ module.exports = function runTransform(webpackProperties, action) {
 	const webpackConfig = Object.keys(webpackProperties).filter(p => {
 		return p !== "configFile" && p !== "configPath";
 	});
-	const initActionNotDefined = action && action !== "init" ? true : false;
+	const addActionDefined = action && action === "add";
```

Comment:
this needs a fallback value and
it will break the init feature as well.

Reply:
Added the fallback value. Actually, we can get away without fallback value as `action` won't be undefined ever. If it's not equal to "add", it `addActionDefined` would false. We will only need fallback value if `action` is undefined. 
Also, I don't see how `init` will break. It will continue to work.